tests/ci/cherry_pick.py

```diff
@@ -473,12 +480,18 @@ def process_backports(self):
 
     def process_pr(self, pr: PullRequest) -> None:
         pr_labels = [label.name for label in pr.labels]
-        if self.must_create_backport_label in pr_labels:
-            branches = [
-                ReleaseBranch(br, pr, self.repo, self.backport_created_label)
-                for br in self.release_branches
-            ]  # type: List[ReleaseBranch]
-        else:
+
+        for label in self.must_create_backport_labels:
+            # We backport any vXXX-must-backport to all branches of the fetch repo (better than no backport)
+            if label in pr_labels or self._fetch_from:
+                branches = [
+                    ReleaseBranch(br, pr, self.repo, self.backport_created_label)
+                    for br in self.release_branches
+                ]  # type: List[ReleaseBranch]
+                found = True
```

Comment:
The `found` is not used

```suggestion
            # We backport any vXXX-must-backport to all branches of the _fetch_from repo (better than no backport)
            if label in pr_labels or self._fetch_from:
                branches = [
                    ReleaseBranch(br, pr, self.repo, self.backport_created_label)
                    for br in self.release_branches
                ]  # type: List[ReleaseBranch]
```